std_msgs/Header header

string turn_light

float32 acceleration
float32 brake
float32 gas
float32 steering

geometry_msgs/Twist center_of_mass

float32 engine_rpm
float32 engine_load
float32 engine_torque
float32 engine_power

int32 current_gear

WheelTelemetries[] wheels